content/token22-in-the-client.md

```diff
@@ -0,0 +1,470 @@
+---
+title: Supporting Token Extension Program from a Client
+objectives:
+- Learn how to effectively integrate multiple Solana token programs within client applications
+- Become proficient in utilizing the SPL TypeScript library for comprehensive token operations
+---
+
+# Summary
+
+- The `Token Extension Program` has all of the same functions as the `Token Program`, with added `extensions`
+- These two token programs: `Token Program` and `Token Extension Program` use seperate addresses and are not directly compatable
+- Supporting both requires simply specifying the correct program ID in client-side functions
```

Comment:
Remove 'simply' - it's an unnecessary adjective. We should add this to the writing guide as well